### ISSUE TITLE
Build step failed with error "no such package '@ros//'"

### ISSUE THREAD

**Issue by rnaidu02** [ApolloAuto/apollo#854]
When I ran the (bash apollo.sh build) command to build Apollo, the following error occurred. I am compiling on a Ubuntu 16.04 system. Please let me know if I am missing any steps. I have followed the steps as outlined in https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_build_and_release.md

Here is the log:

Building: no action running
Building: no action running
Building: no action running
Building: no action running
Building: no action running
[INFO] Building on x86_64...
INFO: Reading 'startup' options from /apollo/tools/bazel.rc: --batch_cpu_scheduling
ERROR: /apollo/modules/third_party_perception/BUILD:24:1: no such package '@ros//': Expected directory at /home/tmp/ros but it does not exist. and referenced by '//modules/third_party_perception:third_party_perception'
ERROR: Analysis of target '//modules/third_party_perception:third_party_perception' failed; build aborted
INFO: Elapsed time: 1.987s
FAILED: Build did NOT complete successfully (5 packages loaded)
============================
[ERROR] Build failed!
[INFO] Took 140 seconds
============================

**Comment by rnaidu02** [comment]
I was able to resolve this issue by performing a step manually:

- extract  /home/tmp/ros-indigo-apollo-1.5.1-x86_64.tar.gz to create the /home/tmp/ros folder. 

Now the build completed without any issues.

**Comment by gengqx** [comment]
You could download the release ros,or you may find a release ros at
```
 /home/tmp/
```
and run this
```
tar zxvf ros-indigo-apollo-1.5.1.x86_64.tar.gz
rsync -av ros/ /apollo/third_party/ros_x86_64
source /apollo/third_party/ros_x86_64/setup.bash
```

**Comment by slivingston** [comment]
based on [the comment above](https://github.com/ApolloAuto/apollo/issues/854#issuecomment-339554293) by the author of the OP, this issue can be closed or subsumed by issue #3605

**Comment by Zhenni17** [comment]
Will close the issue. Please re-open an new issue if there are additional questions.
